### DISCUSSION TITLE
Архитектура проекта

### DISCUSSION THREAD

**Discussion by Metrofanova** [ESEI-ZNU/SortingAlgorithms#3] [General]
**Народ, для описания архитектуры проекта, мне от вас необходимо следующее:**
1.Перечень структур, классов, пользовательских типов с описанием их предназначения и взаимодействия
2.Описание используемых конструкторов, функций ( с приведением из названий, параметров и результата который они возвращают)
3.Описание библиотек которые подключаете для реализации и их использование.

Вопросы, если есть, задавать в эту дискуссию.

**Comment by BihunMykyta** [comment]
УУУ
